### DISCUSSION TITLE
Summit 2023: First Call for Content and Registration Reminder

### DISCUSSION THREAD

**Discussion by john-a-carroll** [delph-in/docs#121] [Announcements]
Dear DELPH-IN Participants,

Petter Haugereid and I are the program chairs for the upcoming DELPH-IN Summit (June 26–30 in A Coruña, Spain). We're looking forward to an exciting and inspiring program of presentations and discussions.

Please let us know what you would like discussed at the summit, by emailing delphinsummit@gmail.com by **Friday, May 19**. Some possibilities are:

* site updates [5–10 minutes, including questions]
* plenary discussions, with introductory problem statement by moderator [typically 10+50 minutes]
* short, problem-focused presentations (possibly backed up by shared paper pre-print) [10+20]
* long presentations (possibly including an extended Q&A) [20+20–30]
* SIG topics
* offers or requests for tutorials

Also, if you're planning to participate (either in person or online), please register - also by **May 19** - by adding your information to this wiki page if you have not already done so:

https://github.com/delph-in/docs/wiki/GaliciaParticipants

To add your information you need to be logged in. Then click the Edit button, go to the bottom of the table, and add your own line like the others, including your name, the dates and times<sup>*</sup> you can attend, and any additional relevant notes. Finally, save your changes and check to see that your addition appears as you intended.

Thanks, and we hope to hear from you soon!

John and Petter

<sup>*</sup> For practical reasons we will be following a "Spanish" schedule, probably starting 10:00 CEST (= 08:00 UTC) and finishing 19:30 CEST (= 17:30 UTC) with a long break for lunch.

**Comment by john-a-carroll** [comment]
A quick reminder about the **May 19** deadline:

If you are expecting to participate in the DELPH-IN Summit (June 26–30 in A Coruña, Spain) and have not yet registered, please add your information to the wiki page: 

[https://github.com/delph-in/docs/wiki/GaliciaParticipants](https://github.com/delph-in/docs/wiki/GaliciaParticipants)

If you have already registered please make sure your attendance status is up to date (i.e. whether in-person or online). Olga, our local host, needs to know the likely number of in-person attendees by May 19, so she can make bookings for the excursion and banquet.

In addition, Petter and I are waiting for your emails (to delphinsummit@gmail.com) offering and/or requesting content. If you want to request a tutorial or suggest a discussion to be led by someone else, we would especially like to know by May 19, so there's sufficient time to find someone suitable.

Thanks!

John and Petter

**Comment by olzama** [comment]
A note from the local organizer: it appears that the online booking system of the NH Collection hotel will tell you there are no rooms available, even if you provide it with the group code. That's not true, there are rooms for us. If you send them an email mentioning the group code, they should be able to book the room for you. Let me know if you need help!

**Comment by olzama** [comment]
Dear @delph-in/participants : 

Just another reminder that the local organizer (me) would really love to have a good estimate for attendance at this point! So, if you planned to register at https://github.com/delph-in/docs/wiki/GaliciaParticipants but forgot, or you forgot to update your status (online vs in-person), please do so today! 

Looking forward to greeting you in A Coruña soon!

**Comment by guyemerson** [comment]
Question for the local organiser: On https://github.com/delph-in/docs/wiki/GaliciaTop it says: "No breakfast, but you don't need it due to the location!"  Does this mean there are nearby cafés with good breakfast?

I'm hoping to book things soon, and to make the whole journey by train :)

**Reply by olzama** [reply]
...with coffee and croissants (etc), for sure, plenty. Not necessarily eggs though (that is just not really done here, although of course in expensive hotels, they probably do it).

**Reply by olzama** [reply]
More info on breakfast in general: https://github.com/delph-in/docs/wiki/GaliciaLocalGuide#coffee-tea-and-breakfast

**Comment by olzama** [comment]
@delph-in/participants !  A very important question *from* the local organizer: how many of those attending in person are vegetarian or have other food restrictions? Please write to me by email if you prefer or indicate here. Please provide this info as soon as possible; the cuisine here is very meat and seafood-heavy so anything beyond that needs to be thought of in advance.

**Comment by john-a-carroll** [comment]
Dear @delph-in/participants

Many thanks to you all for your offers and requests for content. Now that we're well past the May 19 deadline, I hope we have the final list of plenary presentations and discussions – although as usual further suggestions for SIGs and tutorials will be welcome until the Wednesday of the Summit.

We've drawn up a draft schedule for the plenary sessions (minus session start times, which are still to be confirmed). It's in the predictable place:

https://github.com/delph-in/docs/wiki/GaliciaSchedule

This schedule should respect the preferences and constraints you gave us, but please double-check that it will work for you. If it doesn't then please let us know asap.

John & Petter

**Reply by john-a-carroll** [reply]
@delph-in/participants We've finalised the times and chairing arrangements for the plenary sessions. See https://github.com/delph-in/docs/wiki/GaliciaSchedule

The usual start time each day will be 10:30, except on Wednesday when we will start at **10:00** in order to give enough time in the afternoon for the excursion. For those presenting site updates, please note that you have a maximum of 6 minutes – and if you can manage with less that would help since session 1.1 is tightly packed.

Any queries, please email us at delphinsummit@gmail.com or post a comment here. Looking forward to seeing you (in person or virtually) at the summit!

John & Petter

**Comment by EricZinda** [comment]
If it’s not too late I’ll give a Perplexity Site update too.  Forgot that, sorry!

Also, FYI I added the schedule to the docs site: https://delph-in.github.io/docs/summits/GaliciaSchedule/

From: John Carroll ***@***.***>
Date: Tuesday, June 6, 2023 at 2:33 AM
To: delph-in/docs ***@***.***>
Cc: Eric Zinda ***@***.***>, Team mention ***@***.***>
Subject: Re: [delph-in/docs] Summit 2023: First Call for Content and Registration Reminder (Discussion #121)

Dear @delph-in/participants<https://github.com/orgs/delph-in/teams/participants>

Many thanks to you all for your offers and requests for content. Now that we're well past the May 19 deadline, I hope we have the final list of plenary presentations and discussions – although as usual further suggestions for SIGs and tutorials will be welcome until the Wednesday of the Summit.

We've drawn up a draft schedule for the plenary sessions (minus session start times, which are still to be confirmed). It's in the predictable place:

https://github.com/delph-in/docs/wiki/GaliciaSchedule

This schedule should respect the preferences and constraints you gave us, but please double-check that it will work for you. If it doesn't then please let us know asap.

John & Petter
